client/src/components/app/AddLines.js

```diff
@@ -31,24 +33,28 @@ export const AddLines = () => {
 
     // need to add validation
     return (
+        <>
+        <Header />
         <Formik
             // validate={validate}
             initialValues={initValues}
             enableReinitialize={true}
             onSubmit={submit}
         >
             <Form>
-                <label htmlFor="num">Number of Features: </label>
-                <Field name="num" type="number" />
-                <label htmlFor="numVertices">Number of Vertices: </label>
-                <Field name="numVertices" type="number" />
-                <label htmlFor="maxSegmentLength">Max Segment Length: </label>
-                <Field name="maxSegmentLength" type="number" />
-                <label htmlFor="maxSegmentRotation">Max Segment Rotation: </label>
-                <Field name="maxSegmentRotation" type="number" />
+                <strong htmlFor="num">Number of Features: </strong>
+                <Field id="fieldSmall" name="num" type="number" />
```

Comment:
It's best practice to make `id`s unique (e.g. if id `xyz` is used on one element, it should be the only element in the application with that id). However, you can use classes across as many components as you'd like! I'd switch the current `id`s used for styling (e.g. `fieldSmall`, `greenButton`, `boundingBoxList`, etc.) to classes using `className` - React calls it `className` instead of `class`, I forget why. Lmk if you have questions!